src/elements/play-pen/play-pen.ts

```diff
@@ -385,11 +435,12 @@ export class PlayPen extends LitElement {
   }
 
   /** Throttled changes after updating sources. */
-  #setSrcSideEffects = throttle((save: boolean): void => {
+  #setSrcSideEffects = throttle(async (save: boolean): Promise<void> => {
```

Comment:
I think I was getting confused in the other thread. please just zap the async on this since we're no longer awaiting :+1: